frontend/src/metabase/dashboard/components/DashboardHeader/DashboardHeader.tsx

```diff
@@ -301,32 +140,33 @@ export const DashboardHeaderInner = ({
 
   const hasLastEditInfo = dashboard["last-edit-info"] != null;
 
-  const { buttons: headerButtons } = getHeaderButtons();
   const editingButtons = getEditingButtons();
 
   return (
     <>
       <DashboardHeaderComponent
-        headerClassName={CS.wrapper}
         dashboard={dashboard}
         collection={collection}
-        isEditing={isEditing}
         isBadgeVisible={!isEditing && !isFullscreen && isAdditionalInfoVisible}
         isLastEditInfoVisible={hasLastEditInfo && isAdditionalInfoVisible}
-        isEditingInfo={isEditing}
-        isNavBarOpen={isNavBarOpen}
-        headerButtons={headerButtons}
         editWarning={getEditWarning(dashboard)}
         editingTitle={t`You're editing this dashboard.`.concat(
           isHomepageDashboard
             ? t` Remember that this dashboard is set as homepage.`
             : "",
         )}
         editingButtons={editingButtons}
-        setDashboardAttribute={handleSetDashboardAttribute}
         onLastEditInfoClick={() =>
           dispatch(setSidebar({ name: SIDEBAR_NAME.info }))
         }
+        refreshPeriod={refreshPeriod}
```

Comment:
Unfortunately we still have to pass this into `DashboardHeaderComponent`. I'm trying to invert the structure of this component so that the header buttons are actually created within the container, rather than being passed into the container. This will help make the creation of the header component much easier for the future